app/components/List.jsx

```diff
@@ -4,8 +4,19 @@ import React from 'react';
 import ListItem from './ListItem';
 import ReactCSSTransitionGroup from './TimeoutTransitionGroup';
 import { mergeAndPrefix } from '../utils/stylePropable';
+import { List } from 'material-ui';
+
+export default class extends React.Component {
+  static propTypes = {
```

Comment:
No, that is not applying the statics to the constructor, that is applying  statics to the component instance. Take a look at the other components in this project and read FB's React [blog post](https://facebook.github.io/react/blog/2015/01/27/react-v0.13.0-beta-1.html#es6-classes) about how to use ES6 classes.

> Well I think we should stick with static. 

If we were sticking with ES6 classes I would agree. But as soon as this PR gets accepted I will be making the components stamps. Take a look at PR #19.